src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -176,6 +176,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" />
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
+    <ProjectReference Include="..\..\System.Buffers\src\System.Buffers.csproj" />
```

Comment:
I've updated the pull request to use the `project.json` for handling this instead.